src/main/java/org/apache/sysml/api/DMLScript.java

```diff
@@ -102,66 +109,58 @@
+		public DMLScript.RUNTIME_PLATFORM execMode = OptimizerUtils.getDefaultExecutionMode();	// Execution mode standalone, MR, Spark or a hybrid
+		public boolean              gpu           = false;						// Whether to use the GPU
+		public boolean              forceGPU      = false;						// Whether to ignore memory & estimates and always use the GPU
+		public boolean              debug         = false;						// to go into debug mode to be able to step through a program
+		public ScriptType           scriptType    = ScriptType.DML;		// whether the script is a DML or PyDML script
+		public String               filePath 			= null;							// path to script
+		public String 							script				= null;							// the script itself
+		public boolean              help          = false;						// whether to print the usage option
+
+		public final static DMLOptions defaultOptions = new DMLOptions();
+	}
 	
-	public static RUNTIME_PLATFORM rtplatform = OptimizerUtils.getDefaultExecutionMode();
-	public static boolean STATISTICS = false; //default statistics
-	public static int STATISTICS_COUNT = 10;	//default statistics maximum heavy hitter count
-	public static boolean ENABLE_DEBUG_MODE = false; //default debug mode
-	public static boolean USE_LOCAL_SPARK_CONFIG = false; //set default local spark configuration - used for local testing
-	public static String DML_FILE_PATH_ANTLR_PARSER = null;
-	public static ExplainType EXPLAIN = ExplainType.NONE; //default explain
+	public static RUNTIME_PLATFORM 	rtplatform = DMLOptions.defaultOptions.execMode;							// the execution mode
+	public static boolean						STATISTICS = DMLOptions.defaultOptions.stats;									// whether to print statistics
+	public static int 							STATISTICS_COUNT = DMLOptions.defaultOptions.statsCount;			// statistics maximum heavy hitter count
+	public static boolean						ENABLE_DEBUG_MODE = DMLOptions.defaultOptions.debug; 					// debug mode
+	public static ExplainType 			EXPLAIN = DMLOptions.defaultOptions.explainType; 							// explain type
+	public static String						DML_FILE_PATH_ANTLR_PARSER = DMLOptions.defaultOptions.filePath;	// filename of dml/pydml script
+
 	/**
 	 * Global variable indicating the script type (DML or PYDML). Can be used
 	 * for DML/PYDML-specific tasks, such as outputting booleans in the correct
 	 * case (TRUE/FALSE for DML and True/False for PYDML).
 	 */
-	public static ScriptType SCRIPT_TYPE = ScriptType.DML;
-	
-	public static boolean USE_ACCELERATOR = false;
-	public static boolean FORCE_ACCELERATOR = false;
-	
-	// flag that indicates whether or not to suppress any prints to stdout
-	public static boolean _suppressPrint2Stdout = false;
-	
-	public static String _uuid = IDHandler.createDistributedUniqueID(); 
+	public static ScriptType 				SCRIPT_TYPE = DMLOptions.defaultOptions.scriptType;
+
+	public static boolean						USE_ACCELERATOR = DMLOptions.defaultOptions.gpu;
+	public static boolean 					FORCE_ACCELERATOR = DMLOptions.defaultOptions.forceGPU;
```

Comment:
Can you fix the spacing in this section?  It would be better to just use spaces to align everything.